jaeger_client/tracer.py

```diff
@@ -184,6 +197,46 @@ def start_span(self,
 
         return span
 
+    def start_active_span(self,
```

Comment:
This is my bad for not keeping up to tabs for `scopes`, but is it expected for the user to be aware of the concept of `scopes`, and `active spans`? I guess what I'm getting at is would it be better have the existing `start_span` function take care of doing `self.scope_manager.activate(span, finish_on_close)` so that there isn't a costly migration for users?

Reply:
Well this is a part of the opentracing api, and there are tests requiring that particular method to be in place (https://github.com/opentracing/opentracing-python/blob/9ef3e19f101774a53f316ff508ecaeaa4af741bc/opentracing/harness/api_check.py#L70), so it seems to me that the intent is for user to be aware of `scopes`. I don't think that the migration would be costly since one can either continue to pass parent spans to `start_span` and it would continue to work as it used to, or move to using the `start_active_span` and get rid of passing the parent.